core/vm/runtime/pricelist.hpp

```diff
@@ -11,11 +11,15 @@
 namespace fc::vm::runtime {
   using primitives::GasAmount;
   using primitives::TokenAmount;
+  using primitives::sector::AggregateSealVerifyProofAndInfos;
   using primitives::sector::RegisteredPoStProof;
+  using primitives::sector::RegisteredSealProof;
   using primitives::sector::WindowPoStVerifyInfo;
 
   struct Pricelist {
-    Pricelist(ChainEpoch epoch) : calico{epoch >= kUpgradeCalicoHeight} {}
+    // TODO: lotus bug
```

Comment:
Please, describe the bug.

Reply:
old check "epoch >= kUpgradeCalicoHeight" didn't work because of bug, so new check was required.
lotus returned wrong gas prices for negative upgrade height (interopnet starts with hyperdrive upgrade).
https://github.com/filecoin-project/lotus/blob/a577bf340ecceb1b9689872bdb9d6ba637c59c64/chain/vm/gas.go#L212

Reply:
```suggestion
    // TODO: lotus bug
    // old check "epoch >= kUpgradeCalicoHeight" didn't work because of bug, so
    // new check was required. lotus returned wrong gas prices for negative
    // upgrade height (interopnet starts with hyperdrive upgrade).
    // https://github.com/filecoin-project/lotus/blob/a577bf340ecceb1b9689872bdb9d6ba637c59c64/chain/vm/gas.go#L212
```